.github/workflows/mirror.yml

```diff
@@ -20,3 +23,21 @@ jobs:
         github-token: ${{ secrets.GITHUB_TOKEN }}
         source: 'main'
         dest: ${{ matrix.dest_branch }}
+
+  mirror_job_1_x:
+    if: github.ref == 'refs/heads/1.4.x'
+    runs-on: ubuntu-latest
+    name: Mirror main branch to compatible minor version branches
```

Comment:
Mirror 1.4.x branch to compatible minor version branches

Reply:
Done